openpype/hosts/max/plugins/publish/collect_review.py

```diff
@@ -19,30 +22,40 @@ class CollectReview(pyblish.api.InstancePlugin,
 
     def process(self, instance):
         nodes = instance.data["members"]
-        focal_length = None
-        camera_name = None
-        for node in nodes:
-            if rt.classOf(node) in rt.Camera.classes:
-                camera_name = node.name
-                focal_length = node.fov
 
+        def is_camera(node):
+            is_camera_class = rt.classOf(node) in rt.Camera.classes
+            return is_camera_class and rt.isProperty(node, "fov")
+
+        # Use first camera in instance
+        cameras = [node for node in nodes if is_camera(node)]
+        if cameras:
+            if len(cameras) > 1:
+                self.log.warning(
+                    "Found more than one camera in instance, using first "
+                    f"one found: {cameras[0]}"
+                )
+            camera = cameras[0]
+            camera_name = camera.name
+            focal_length = camera.fov
+        else:
+            raise KnownPublishError(
+                "Invalid object found in 'Review' container."
+                " Only native max Camera supported"
+            )
```

Comment:
This isn't necessarily true - there might no objects found in the Review container at all. Right?

So should likely be something like this?
```suggestion
            raise KnownPublishError(
                "Unable to find a valid camera in 'Review' container."
                " Only native max Camera supported. "
                f"Found objects: {members}"
            )
```